source/code_of_conduct.html.haml

```diff
@@ -66,49 +52,38 @@ title: RubyConf - Colombia
 
           %p Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe. In the context of in-person events, organizers will also provide escorts as desired by the person experiencing distress.
 
         .accordion__header
           %h2.accordion__header--title 7. Addressing Grievances
         .accordion__body
-          %p If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify GOVERNING_BODY with a concise description of your grievance. Your grievance will be handled in accordance with our existing governing policies. [LINK_TO_POLICY]
+          %p If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify us with a concise description of your grievance. Your grievance will be handled in accordance with our existing governing policies. (See Contact Info section)
 
-          %p [NOTE: Every organization's governing policies should dictate how you handle warnings and expulsions of community members. It is strongly recommended that you mention those policies here or in Section 7 and that you include a mechanism for addressing grievances.]
+          /%p [NOTE: Every organization's governing policies should dictate how you handle warnings and expulsions of community members. It is strongly recommended that you mention those policies here or in Section 7 and that you include a mechanism for addressing grievances.]
 
         .accordion__header
           %h2.accordion__header--title 8. Scope
         .accordion__body
-          %p We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues--online and in-person--as well as in all one-on-one communications pertaining to community business.
-          
-          %p This code of conduct and its related procedures also applies to unacceptable behavior occuring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.
+          %p We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues—online and in- person—as well as in all one-on-one communications pertaining to community business.
 
         .accordion__header
-          %h2.accordion__header--title 9. Contact info
+          %h2.accordion__header--title 9. Contact Info
         .accordion__body
-          %p [YOUR CONTACT INFO HERE -- this should be a single person or a small team who can respond to issues directly]
+          %p
+            %a{:href => "mailto:rubyconfco@gmail.com"} Governing Body
```

Comment:
yep we own it. We could create `organizers@rubyconf.co` but I don't have domain access